.idea/jarRepositories.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you remove this file from your git commit? This is .idea file and not part of vcs